lib/runner/cli/cli.js

```diff
@@ -469,8 +469,11 @@ class CliRunner {
         let promise = Promise.resolve();
 
         if (this.test_settings.selenium && this.test_settings.selenium.start_process) {
-          const SeleniumServer = require('../../transport/selenium-webdriver/selenium.js');
-          this.seleniumService = SeleniumServer.startServer(this.test_settings);
+          const Server = this.test_settings.selenium.use_appium
```

Comment:
we should use a factory here.

Reply:
You mean `factory` as variable name or use a `TransportFactory` class method to decide whether appium is being used or not?

Reply:
use a factory to create the server